app/(tabs)/summary/main-summary.tsx

```
@@ -3,13 +3,13 @@ import {useLocalSearchParams} from 'expo-router';
 import {Button, Text} from 'react-native-paper';
 
-import {BarChart, Chip, DatePicker} from '@/components';
+import {BarChart, Chip, DatePicker, PieChartBar} from '@/components';
 import {lastDayOfMonth} from 'date-fns';
 import {useAppSelector} from '@/hooks';
 import {aggregateExpenses} from '@/redux/main/selectors';
-import {EXCLUDED_CAT, formatPrice} from '@/common';
-import {parseInt} from 'lodash';
-import {SafeAreaView} from 'react-native-safe-area-context';
+import {EXCLUDED_CAT, formatPrice, shortenText} from '@/common';
+import _, {parseInt} from 'lodash';
```

Biome

[error] Do not shadow the global "parseInt" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)